CONTRIBUTING.md

```diff
@@ -1,62 +1,152 @@
-# Contributing
+# Contributing To Simple Icons
 
-This repository welcomes contributions and corrections. Before submitting a pull request, ensure that you respect these guidelines for SVGs:
+Simple Icons welcomes contributions and corrections. Before contributing, please make sure you have read the guidance below.
```

Comment:
... you have read the ~guidelies~ guidelines below.

Reply:
Got it.